playbook/app/pb_kits/playbook/pb_table/index.ts

```diff
@@ -99,6 +100,34 @@ export default class PbTable extends PbEnhancedElement {
       });
     }
 
+    handleCollapsibleRow() {
+      if (window.isCollapsibleRowHandled) return;
+
+      const collapsibleRows = document.querySelectorAll('.pb_table_collapsible_row');
+      if (collapsibleRows.length > 0) {
+        collapsibleRows.forEach((row) => {
+          const previousRow = row.previousElementSibling;
+
+          if (
+            previousRow &&
+            previousRow.tagName === 'TR'
+          ) {
+            const tdCount = previousRow.querySelectorAll('td').length;
+            console.log(`Number of <td> elements in the previous <tr>: ${tdCount}`);
+
+            const collapsibleTd = row.querySelector('td');
+            if (collapsibleTd) {
+              collapsibleTd.colSpan = tdCount;
+            }
+          } else {
+            console.log('Previous row does not match collapsible conditions.');
```

Comment:
This looks great @nickamantia! But lets remove these console logs, otherwise we'll be getting a lot of logs in Nitro when devs go to use this!

Reply:
yes will do thanks!

Reply:
done!